src/docs/config/overview.md

```diff
@@ -166,6 +167,30 @@ The `preamble` configuration is a `string` that represents a preamble in the mai
 preamble: 'Built with Stencil'
 ```
 
+## sourceMap
+
+*default: `false`*
+
+When set to `true`, sourcemaps will be generated for a project.
+
+```tsx
+sourceMap: true
+```
+
+Sourcemaps create a translation between Stencil components that are written in TypeScript & JSX and JavaScript that is
+output by Stencil. They allow for a better debugging experience for Stencil components, in that they allow external
```

Comment:
```suggestion
output by Stencil. Enabling source maps in your project allows for an improved debugging experience for Stencil components. For example, they allow external
```

Reply:
This sentence is meant to describe sourcemaps generally, rather than the Stencil specific aspects. The idea here is you shouldn't have to go google 'sourcemap' to figure out what one is (at a high level at least)